globals.go

```diff
@@ -1096,7 +1096,7 @@ const (
 // Validate checks that the container OS is recognized.
 func (c ContainerOS) Validate() error {
 	switch c {
-	case LinuxOS, WindowsOS:
+	case LinuxOS, WindowsOS, "":
```

Comment:
I'm not sure I understand - these properties are only set on a container itself, not on a task or variant. There are other validations checking if a task / variant's run_on is referencing a valid list of containers or distros.

Reply:
It wouldn't be possible for a host task/variant to have this set.

Reply:
Sorry, I can rephrase this - I meant that an empty OS is not really a valid OS for a container definition since we don't have a default OS. We should expect `(model.ContainerSystem).Validate` to error for an empty OS. 

In any case, it seems like the empty string validation went away in a newer commit.